### ISSUE TITLE
Is this app still under developement?

### ISSUE THREAD

**Issue by qns7** [onebadidea/swiftquit#41]
I would love to know from the developer if this app is still under contruction or if the project is basically dead?
Best regards Q

**Comment by aayushbhatia06** [comment]
Same question

**Comment by haznai** [comment]
No, I would suggest replacing this wonderful (but not actively maintained software) with supercharge: https://sindresorhus.com/supercharge

when compared with swiftquit, supercharge has following tradeoffs:

**pros**
- actively maintained
- supporting a great open source maintainer
- lots of functionalities (for some, might be a con)

**cons**
- not open source
- costs money
- lots of functionalities (for some, might be a pro)

**Comment by arcticfoxwpg** [comment]
> No, I would suggest replacing this wonderful (but not actively maintained software) with supercharge: https://sindresorhus.com/supercharge
> 
> when compared with swiftquit, supercharge has following tradeoffs:
> 
> **pros**
> 
> * actively maintained
> * supporting a great open source maintainer
> * lots of functionalities (for some, might be a con)
> 
> **cons**
> 
> * not open source
> * costs money
> * lots of functionalities (for some, might be a pro)

It looks like it does not do the same thing as Swiftquit? (ie. red X closes the app, yellow - minimizes.)

**Comment by Martins6** [comment]
I'd use Claude Code or an other AI Coding to fix any issues that come up with the repo since this is open source. Not ideal, not as fun, but...better than paying money to have such a simple functionality

**Comment by Ahmet-Dedeler** [comment]
@Martins6 if you did make the changes, would you mind creating a github repo for it? would help a lot 🙏

**Comment by weiShaoY** [comment]
> No, I would suggest replacing this wonderful (but not actively maintained software) with supercharge: [sindresorhus.com/supercharge](https://sindresorhus.com/supercharge)
> 
> when compared with swiftquit, supercharge has following tradeoffs:
> 
> **pros**
> 
> * actively maintained
> * supporting a great open source maintainer
> * lots of functionalities (for some, might be a con)
> 
> **cons**
> 
> * not open source
> * costs money
> * lots of functionalities (for some, might be a pro)

This does not support Chinese.
